src/hyppo/freeze.asm

```diff
@@ -887,19 +917,38 @@ dfp1:   lda $d680,x
         ;; since it gets stomped while saving palettes
         lda $d070
         sta freeze_d070
+        ;; Also save $D08F (VIC IV Errata)
+        ;; We can't extend the VIC IV DMA entry due to
+        ;; it spanning $D081 which must not be written to
+        ;; lest the SD reading is disturbed.
+        lda $d08f
+        sta freeze_vic_errata
         rts
 
+version_sentinel: !text version_sentinel_str
 freeze_scratch_area:
         !8 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
 freeze_d070:
         !8 0
+freeze_version:
+        !text version_sentinel_str
+        ;; *** NOTE: Update the version below if more values are added to be restored from
+        ;; the scratch area.  This prevents us restoring values that were not
+        ;; actually frozen but just happened to be in a freeze sector when an
+        ;; older freeze was made.
+        !8 1
+freeze_vic_errata:
+        !8 0
+        ;; See NOTE above ;)
+freeze_scratch_area_end:
```

Comment:
isn't it better to make this area bigger, so further changes won't change the general markup of this extra section? So bump it up to be 32 bytes in total?

Reply:
The note is about updating the version if more stuff is added to this section.  I'm trying to avoid future extensions of this area (which save bytes that come from disparate parts of memory) from restoring to locations that were not being saved when the freeze was created.

Saving more now (say filling with zeroes) doesn't avoid this problem.  We could end up unfreezing zeroes to memory even though they were not actually being saved when that older freeze was made.

This is definitely adding some complication that may be premature if we don't care about preserving backward compatibility of freezes.  I can also understand if you'd like to put more thought into a comprehensive way to version the freezes -- which may exist and I don't even know about.

Sorry this is getting long :) But I just want to mention that I was mimicking what was done to save $D070 (I think it was).  Another option is to add an entire sector just to save $D08F, but that feels excessive.